modules/svg/image_loader_svg.cpp

```diff
@@ -68,10 +70,12 @@ void ImageLoaderSVG::create_image_from_string(Ref<Image> p_image, String p_strin
 		return;
 	}
 	picture->viewbox(nullptr, nullptr, &fw, &fh);
-	ERR_FAIL_COND(Math::is_zero_approx(p_scale));
+
+	// FIXME: p_upsample is not used.
```

Comment:
Could be as simple as multiplying it to `fw * p_scale` and `fh * p_scale` below as done in the previous loader, but should be checked.